definitions/infra-azureappliactioninsight/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: INFRA
+type: AZUREAPPLICATIONINSIGHT
```

Comment:
Minor typo on the folder name here: s/infra-azureappliactioninsight/infra-azureapplicationinsight/g